lib/provisioning/account.js

```diff
@@ -47,118 +57,118 @@ function createSubAccount(name, cloudName, customAttributes, enabled, baseAccoun
 }
 
 /**
- * @desc - Delete a subAccount by subAccountID
- * @param subAccountId {string} - The ID generated when using createSubAccount
+ * @desc - Delete the specified sub-account. Supported only for accounts with fewer than 1000 images.
```

Comment:
What happens when I try to delete a larger sub-account?
BTW, it says "supported only for **accounts**..." meaning, if I have an account with 2 sub-accounts - one with >1000 images and one with 3 images, I still can't delete the small sub-account?

Reply:
Won't be a blocker for this PR, but let's figure it out and update the docs if needed